recipes/wildmeshing/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "wildmeshing" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wildmeshing/wildmeshing-python/archive/{{ version }}.tar.gz
+  sha256: 742154f0f705eb7bf39c5c65c6cba94196c19c411e60ef2eb13152aa4341f415
+
+build:
+  number: 0
+  skip: True  # [py<31]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - pip
+    - libblas
+    - libcblas
+    - numpy
+    - gmp     # [unix]
+    - mpir    # [win]
+  run:
+    - python
+    - mpir    # [win]
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - wildmeshing
+
+about:
+  home: https://github.com/wildmeshing/wildmeshing-python
+  license: LGPL
+  license_family: LGPL
+  license_file: LICENSE
+  summary: 'Simple, fast, robust meshing'
+
+  description: 'Simple, fast, robust meshing'
+  doc_url: https://github.com/wildmeshing/wildmeshing-python
+  dev_url: https://github.com/wildmeshing/wildmeshing-python
+
+extra:
+  recipe-maintainers:
+    - teseoch
+    - Yixin-Hu
```

Comment:
@yixin-hu are you ok being a maintainer here?

Reply:
Yes.

Reply:
Great!